src/mock-dnas/holofuel.js

```diff
@@ -332,17 +297,28 @@ function listPending ({ origins }) {
   throw new Error('Array value for origins param of list_pending is not supported in the mock dna')
 }
 
+function receivedPaymentsHashMap (promises) {
+  return promises.forEach(promise => {
```

Comment:
`.forEach` always returns `undefined`, so this function always returns `undefined`. `.map` will return an array that is the result of applying the function to each item in the original array. But that will still do something a little strange in this case. If you used `.map` the result of this function would be an array of objects, each of which has exactly one key.

`receivedPaymentsHashMap([1, 2, 3])` 
->
```
[{
  1: {Ok: { result: /* hashed string of '1accepted' */} }
},
{
  2: {Ok: { result: /* hashed string of '2accepted' */} }
}, 
{
  3: {Ok: { result: /* hashed string of '3accepted' */} }
}]
```

But perhaps that's what you want?